webapp/src/components/table/tableGroupHeaderRow.tsx

```diff
@@ -127,12 +127,12 @@ const TableGroupHeaderRow = React.memo((props: Props): JSX.Element => {
                                         onClick={() => mutator.deletePropertyOption(boardTree, boardTree.groupByProperty!, group.option)}
                                     />
                                     <Menu.Separator/>
-                                    {Constants.menuColors.map((color) => (
+                                    {Object.entries(Constants.menuColors).map(([id, color]) => (
```

Comment:
SUPER NIT!
Can we change this to `key` to make it more inline with the wording of `key / value`  cause I didn't immediately know `id` was the `key` but rather I was thinking `id` is a property of the object

Reply:
This also falls more in line with docs wording of what is returned `[key , value]` ( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#parameters )

Reply:
Yes, you are right, sounds more correct. Thanks! :)